### ISSUE TITLE
Makefile lacks the HEADERS_COMPAT variable

### ISSUE THREAD

**Issue by gdm413229** [farhaven/yadex#3]
Inspected the makefile with nano and found out that there are variables pointing to the headers of everything else excluding the compatibility functions.  Any workaround is appreciated, even a Linux-specific makefile is golden, due to the fact that GNU libc lacks the strlcpy and strlcat functions.

**Comment by farhaven** [comment]
Yeah, that's been in my "fix me" pipeline for a while. Will fix tomorrow. That will likely also fix #2.

**Comment by farhaven** [comment]
Can you try HEAD again? I think that should fix your build.

**Comment by gdm413229** [comment]
Can you tell what HEAD is???

**Comment by farhaven** [comment]
Yeah, sorry. It's the current top commit on the `master` branch
